# decision robot has taken from the current state
string decision
# expected battery life at the next interval
uint32 battery_life
# expected reward at the current interval
float32 reward
# actual battery state at current interval
uint32 current_battery
# current time 
time time

